### PR TITLE
ghostscript: update to 10.0.0

### PR DESCRIPTION
**Pull request by clyhtsuriva**
#### Testing the changes
- I tested the changes in this PR: **briefly**

#### Local build testing
- I built this PR locally for my native architecture, x86_64
- I crossbuilded this PR locally for these architectures:
  - armv7l
  - armv6l
  - aarch64-musl

### CODE REVIEW

File: srcpkgs/ghostscript/template [hunk 1]
```diff
@@ -12,7 +12,7 @@ license="AGPL-3.0-or-later, CPL-1.0"
 homepage="https://www.ghostscript.com/"
 changelog="https://www.ghostscript.com/doc/${version}/News.htm"
```
Comment (classabbyamp) [inline srcpkgs/ghostscript/template:13]:
```suggestion
changelog="https://ghostscript.readthedocs.io/en/latest/News.html"
```

File: srcpkgs/ghostscript/template [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Template file for 'ghostscript'
 pkgname=ghostscript
-version=9.55.0
+version=9.56.1
```
Comment (classabbyamp) [inline srcpkgs/ghostscript/template:3]: 10.0.0 is available

**Comment by clyhtsuriva** [comment]
```
=> ERROR: libgs-10.0.0_1: SONAME bump detected: libgs.so.9 -> libgs.so.10
=> ERROR: libgs-10.0.0_1: please update common/shlibs with this line: "libgs.so.10 libgs-10.0.0_1"
=> ERROR: libgs-10.0.0_1: all reverse dependencies should also be revbumped to be rebuilt against libgs.so.10:
=> ERROR:    ghostscript-9.55.0_1
=> ERROR:    gimp-2.10.30_1
=> ERROR:    libmagick-7.1.0.48_1
=> ERROR:    libmagick6-6.9.12.12_2
=> ERROR:    libspectre-0.2.10_1
=> ERROR: libgs-10.0.0_1: cannot continue with installation!
=> ERROR: libgs-10.0.0_1: pre-pkg_99-pkglint: 'grep -E "${_pattern}" $mapshlibs' exited with 1
=> ERROR:   in hook() at common/hooks/pre-pkg/99-pkglint.sh:154
=> ERROR:   in run_func() at common/xbps-src/shutils/common.sh:21
=> ERROR:   in run_pkg_hooks() at common/xbps-src/shutils/common.sh:262
=> ERROR:   in main() at common/xbps-src/libexec/xbps-src-prepkg.sh:47
```

**Comment by classabbyamp** [comment]
you'll have to update it in `common/shlibs` and revbump every package that uses that shlib (e.g. `xrevshlib ghostscript | xargs xrevbump 'bump for ghostscript-10.0.0'`)

**Comment by clyhtsuriva** [comment]
```
configure: error: No JBIG2 decoder available, required for PDF support
configure: error: Recursive call to configure script failed
=> ERROR: ghostscript-10.0.0_1: do_configure: './configure ${configure_args} --enable-dynamic --with-ijs --with-jbig2dec --with-omni --with-x --with-drivers=ALL --with-fontpath=/usr/share/fonts/Type1:/usr/share/fonts --enable-fontconfig --enable-freetype --enable-openjpeg --with-libpaper --without-luratech --without-omni --with-system-libtiff --disable-compile-inits $(vopt_enable cups)' exited with 1
=> ERROR:   in do_configure() at srcpkgs/ghostscript/template:42
```

**Comment by clyhtsuriva** [comment]
new local issue while trying to install ghostscript :
```
libmagick-7.1.0.48_1: broken, unresolvable shlib `libgs.so.9'
libspectre-0.2.10_1: broken, unresolvable shlib `libgs.so.9'
```

**Pull request closed without merging**